browser/ui/views/frame/vertical_tab_strip_region_view.cc

```diff
@@ -10,45 +10,50 @@
 
 #include "brave/browser/ui/brave_browser.h"
 #include "brave/browser/ui/tabs/brave_tab_prefs.h"
+#include "brave/browser/ui/tabs/brave_tab_prefs.h"
```

Comment:
nit: oops it was already included :)

Reply:
Ah yeah, I seemed to be messed up with rebasing haha thanks